man/systemd.unit.xml

```diff
@@ -192,9 +192,14 @@
     over unit files wherever located. Multiple drop-in files with different names are applied in
     lexicographic order, regardless of which of the directories they reside in.</para>
 
-    <para>Service units also support a top-level drop-in directory for modifying the settings of all service units. See
-    <citerefentry><refentrytitle>systemd.service</refentrytitle><manvolnum>5</manvolnum></citerefentry>
-    for details.</para>
+    <para>Units also support a top-level drop-in with <filename>service.d/</filename>
```

Comment:
`... <filename><replaceable>type</replaceable>.d/<filename>, where <replaceable>type</replaceable> may be e.g. <literal>service</> or <literal>socket</>`